refactor(conf): delete migrations for deprecated fields (#1439)
lingui/js-lingui: website/docs/releases/migration-4.md

```

Read more about custom extractor on the [Advanced: Custom Extractor](/guides/custom-extractor) page.

> ### Configuration migrations for deprecated options were deleted.

Reviewer: ```suggestion
### Configuration migrations for deprecated options were deleted
```

> Migration for older

Reviewer: ```suggestion
Migration for the following older options:
```

>  - `LinguiConfig.localeDir`,
>  - `LinguiConfig.srcPathDirs`,
>  - `LinguiConfig.srcPathIgnorePatterns`
>  - `LinguiConfig.fallbackLocale`

Reviewer: ```suggestion
 - `localeDir`,
 - `srcPathDirs`,
 - `srcPathIgnorePatterns`,
 - `fallbackLocale`
```

> Were deleted from source code. This should affect only users who migrate v2 -> v4 directly.

Reviewer: ```suggestion
were deleted from the source code. This should affect only users who are migrating from `v2` to `v4` directly.
```
